lib/balance-tx/lib/internal/Internal/Cardano/Write/Tx/Balance.hs

```diff
@@ -285,6 +288,9 @@ import qualified Cardano.Api.Byron as Cardano
 import qualified Cardano.Api.Shelley as Cardano
 import qualified Cardano.CoinSelection.UTxOIndex as UTxOIndex
 import qualified Cardano.CoinSelection.UTxOSelection as UTxOSelection
+import Cardano.Ledger.BaseTypes
+    ( StrictMaybe (..)
+    )
```

Comment:
wrong section

Reply:
Fixed by c1f318b260a9506ce744a5217f274c9a05feafdf. (Feel free to cherry-pick.)